grails-bootstrap/src/main/groovy/org/grails/io/support/SpringIOUtils.java

```diff
@@ -423,12 +423,12 @@ private static SAXParserFactory createParserFactory() throws ParserConfiguration
                 // ignore, parser doesn't support
             }
             try {
-                saxParserFactory.setFeature("http://apache.org/xml/features/nonvalidating/load-dtd-grammar", false);
+                saxParserFactory.setFeature("https://apache.org/xml/features/nonvalidating/load-dtd-grammar", false);
             } catch (Exception e) {
                 // ignore, parser doesn't support
             }
             try {
-                saxParserFactory.setFeature("http://apache.org/xml/features/nonvalidating/load-external-dtd", false);
+                saxParserFactory.setFeature("https://apache.org/xml/features/nonvalidating/load-external-dtd", false);
```

Comment:
I don't think these strings should be changed. Aren't they keywords for turning on/off features of the parser.